gridsome/lib/webpack/createServerConfig.js

```diff
@@ -11,7 +11,6 @@ module.exports = async app => {
   config.entry('app').add(resolve('../../app/entry.server.js'))
 
   config.target('node')
-  config.externals([/^(vue|vue-router|vue-meta)$/])
```

Comment:
Making these external causes `vue-server-renderer` to require them, which is wrong as it doesn't declare them as dependencies.
I couldn't figure out how to make `vue-server-renderer` require them on behalf of `gridsome` so it removed for now

Reply:
I don't think it will be a problem to include them in the server bundle, for now. I've been working on implementing Vue 3 lately and made a simple bundle renderer where the [resolver](https://github.com/vuejs/vue/blob/dev/src/server/bundle-renderer/create-bundle-runner.js#L53-L65) is run within `gridsome`. That should make it possible to have them as externals again, I think.

And thanks for the PR. I'll take a closer look one of the next days!

Reply:
It seems like `createBundleRenderer` from `vue-server-renderer` can take a `basedir` option. Which will make `vue-server-renderer` resolve the exernal from that dir instead. You can try to add `basedir: __dirname` [here](https://github.com/gridsome/gridsome/blob/master/gridsome/lib/server/createRenderFn.js#L19-L24) and see if it helps.

Reply:
Ah there it is, that works. :+1: